tutorials/Augurpy.ipynb

```diff
@@ -0,0 +1,195 @@
+{
```

Comment:
<p>Let's start with some introduction. What is Augur? What is Augurpy? What does it do? What is the dataset that we will use here for demonstration?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/augurpy/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>